### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by PhaserlockBren** [jjburton/cgmTools#247]
Ability to save out sub anim list assignments and import into another scene

**Comment by jjburton** [comment]
This is a good idea. Lemme think about how to do it.

**Comment by jjburton** [comment]
@PhaserlockBren | Functionally what do you think of a right click menu in Scene to generate a shotList from?

**Comment by PhaserlockBren** [comment]
That could work. You mean - Basically a right click / import shotlist type of functionality?

**Comment by jjburton** [comment]
Yeah. In whatever scene you're in. Rightclick the one you want in Scene and pull that. I'd have to do it via meta data to start. Otherwise you have to open the other file to get it.
